experimental/dds/ot/ot/package.json

```diff
@@ -72,7 +72,7 @@
 		"prettier:fix": "prettier --write . --cache --ignore-path ../../../../.prettierignore",
 		"test": "npm run test:mocha",
 		"test:coverage": "c8 npm test",
-		"test:mocha": "npm run test:mocha:cjs && npm run test:mocha:esm",
+		"test:mocha": "npm run test:mocha:esm",
```

Comment:
Can we instead of just eliminating cjs from script, add an echo with note. Something like
```
"test:mocha": "npm run test:mocha:esm && echo skipping cjs to avoid overhead - npm run test:mocha:cjs"
```